Mjml.Net/MjmlOptions.cs

```diff
@@ -83,5 +83,10 @@ public sealed record MjmlOptions
         /// The current validator.
         /// </summary>
         public IValidatorFactory? ValidatorFactory { get; init; }
+
+        /// <summary>
+        /// The loader for included texts.
```

Comment:
@SebastianStehle, maybe the summary could be changed to `The file path loader for <mj-include path="..." type="..."> which handles loading the files from the specified path attribute. `